vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -31596,3 +31644,40 @@ THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDE
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "QUDT Unit of Measure Vocabulary Metadata Version 2.1.35" ;
 .
+
+unit:KiloM2 a qudt:Unit;
```

Comment:
This is already present.

Reply:
rebased onto main and removed the duplicate unit:KiloM2